content/en/tests/setup/dotnet.md

```diff
@@ -300,488 +290,6 @@ A test can optionally have additional information like:
 
 Create tests in a suite by calling `suite.CreateTest()` and passing the name of the test. When a test ends, call `test.Close()` with one of the predefined statuses.
 
-### API interface
```

Comment:
Do we have somewhere else where the API reference would live? Maybe a markdown in our dd-trace-dotnet repo?

Reply:
I don't think it's necessary personally. None of the other languages have it. I would bet large amounts of money that noone has ever looked at it. Especially as noone has ever actually used the public API 😉